index.html

```diff
@@ -25,7 +25,40 @@
                 <img class="menu__icon" src="./images/Header/icons/hamburger.svg" alt="Menu icon" />
             </div>
         </header>
-    </div>
+
+        <div class="hero container">
+            <div class="hero__content">
+                <div class="hero__text">
+                <span class="hero__label">Hello</span>
+                <h1 class="hero__title">I’m <span class="hero__title--name">Sarah Jackson</span></h1>
+                <h2 class="hero__subtitle">A UX / UI designer, product designer, content creator and creative director.</h2>
+                <p class="hero__text">Of course, there’s so much more to me than just a few fancy titles. Scroll down and get to know me,</p>
+                </div>
```

Comment:
Seems like you've given .hero__text to both the div and the p inside .hero__content Since that class has a margin-top of 20px the section also has that margin, even tho it shouldn't.
![image](https://user-images.githubusercontent.com/68795596/202477292-ee773c59-1d77-4aeb-bd3e-7fd54c7aac2b.png)